src/include/dns-server/dialog-main.rb

```diff
@@ -40,6 +40,9 @@ def initialize_dns_server_dialog_main(include_target)
       @initial_screen = "start_up"
 
       @service = SystemdService.find("named")
+
+      raise "Service 'named' not found. Please, install the required packages." unless @service
+
```

Comment:
OK, makes sense, thanks for explanation. So basically in the current state of code we will never throw this exception.